templates/gallery/gallery-inner.amp.html

```diff
@@ -0,0 +1,207 @@
+              <p class="gallery-base-invert relative z1 mb2">
+                Suspendisse quisque suscipit mauris ac culpa. Duis etiam mauris. Lacinia volutpat ut. Congue etiam lobortis. Ultrices aliquam rutrum. Dignissim fermentum odio amet id tellus. At taciti imperdiet. Consequat ut blandit. Amet justo posuere. Metus ligula amet. Vehicula libero enim. Lectus sed elit ante dui quis leo nobis.
+              </p>
+            </div>
+            <div class="col col-12 md-col-4">
+              <amp-img
+                width="8"
+                height="12"
+                layout="responsive"
+                src="../../img/gallery/skyline.jpg"
+                srcset="
+                  ../../img/gallery/skyline.jpg 1x,
+                  ../../img/gallery/skyline@2x.jpg 2x
+                "
+                alt="mountain at sunset"
+              ></amp-img>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="gallery-base-section gallery-base-rich-text">
+        <div class="gallery-base-container mx-auto">
+          <div class="clearfix">
+            <div class="col col-12">
+              <h2 class="gallery-base-invert relative z1 mb2">About the Artist</h2>
+            </div>
+            <div class="col col-12 md-col-7">
+              <p class="gallery-base-invert relative z1 mb2">
+                Velit sed quis. Tempus in sed integer morbi lacus ac luctus elit phasellus elementum gravida. Dui at consectetuer. Est in pellentesque. Consequat sapien consectetuer. Nam lacinia erat mollis sodales in magna nec sapien ultrices scelerisque quam.
+              </p>
+              <p class="gallery-base-invert relative z1 mb2">
+                Ultrices nunc dolor voluptates euismod morbi amet in mi libero elit malesuada. Fringilla potenti magna. Orci ipsum est ac nunc eget maecenas ligula dui. Eu a suspendisse. Neque at volutpat. Nam ut ligula. Cras turpis urna consequuntur varius tempus aenean maecenas purus sem faucibus facilisis penatibus volutpat proin. Wisi libero in. Venenatis tellus elementum.
+              </p>
+            </div>
+            <div class="col col-12 md-col-5">
+              <amp-img
+                width="400"
+                height="400"
+                class="gallery-inner-offset-img"
+                layout="responsive"
+                src="../../img/gallery/lake.jpg"
+                srcset="
+                  ../../img/gallery/lake.jpg 1x,
+                  ../../img/gallery/lake@2x.jpg 2x
+                "
+                alt="blue lake against skyline"
+              ></amp-img>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="gallery-base-section gallery-base-rich-text">
+        <div class="gallery-base-container mx-auto">
+          <h2 class="gallery-base-invert relative z1 pb2 mb2">Other Exibits</h2>
+          <ul class="related-section-items list-reset flex flex-wrap m0">
```

Comment:
namespace the class

Reply:
I believe you did not want the entire `ampstart-related-section` is that why you are not using it here?

Reply:
response?